psydac/core/bsplines.py

```diff
@@ -603,6 +606,9 @@ def make_knots( breaks, degree, periodic ):
     periodic : bool
         True if domain is periodic, False otherwise.
 
+    multiplicity: int
+        multiplicity of the knots in the knot sequence
```

Comment:
You should add "we assume that the same multiplicity applies to each interior knot"

Moreover, you should check that `1 <= multiplicity <= degree + 1`

Reply:
done